### DISCUSSION TITLE
Bed of excelsior doesn't set fire

### DISCUSSION THREAD

**Discussion by bjmvandongen** [firemodels/fds#13002] [Q&A]
Hi, 

For a validation case, I am reproducing a fire experiment on a siporex board with excelsior. I specified the excelsior with lagrangian particles. The problem I have now is that the the particles do not burn. The simulation gave no error and FDS was completed successfully. Can someone look at my input file to see what is wrong with it?

[excelsiorflat.txt](https://github.com/firemodels/fds/files/15471468/excelsiorflat.txt)

**Comment by drjfloyd** [comment]
Your ignition fire lasts for only 11 s.  Look at the particle temperatures and see if your ignition source is heating your particles to their ignition temperature.

**Reply by bjmvandongen** [reply]
I increased the HRRPUA of the ignitor  to 1460 kW/m^2 and ran the simulation for 150 seconds. The particles reach a temperature of around 1500 degrees celsius. However, after 12 seconds, there is no fire anymore (see figure). 

<img width="461" alt="no_ignition" src="https://github.com/firemodels/fds/assets/169988139/acc244dc-6301-4d37-9de8-62aad320e006">
[vegetation_model.txt](https://github.com/firemodels/fds/files/15481929/vegetation_model.txt)

**Reply by drjfloyd** [reply]
You are trying to predict the burning rate of a fire. This is research and you need to take the time to delve into the details to understand what is happening in your simulation. We cannot do your research for you. 

Have you tested your material properties to ensure they give appropriate burning rates for your needs?
Is your grid resolution fine enough for this simulation?  You are putting ~0.8 kg of excelsior over 21 m2 of area. This is 40 g/m2. This is not going to be an intense fire and with a 10 cm grid the resolved fire may not be providing enough heat to keep the fire going.

**Reply by bjmvandongen** [reply]
The last five weeks I have been delving into this simulation. I have tested a lot of things.
- I tested the simulation with a finer grid (2x2x2 cm^3): now the fire started a little bit but went out after 30 sec. So grid size is not the (only) problem here.
-  I have tested every parameter in the pyrolysis model (I took half the value and doubled it) and none had a big influence, except for different A and E values for the pyrolysis. Then, the fire sustained a little bit longer but still went out before reaching the end. The fire also became smaller towards the end. (with these other A and E values, the fire went too fast).
- I also performed a TG analysis and with those new A and E values, the fire did not progress.
- I also increased the amount of particles per cell. That did have an influence but the fire still went out.
- I also tested the simulation with the default sphere drag law and that had a bit of an influence, but still the fire went out.
- I also increased the domain size to see if that would have an influence, but that did not matter. I also looked at the fuel vapor and oxygen fractions in slices in the center of the bed (PBY=0) and it seems that there is enough oxygen and the fuel vapor is not leaving the domain. 
- The only parameters that did make a sort of large impact were the fuel density, bulk density and surface to volume ratio. But these are all parameters I cannot change since they are mentioned in the article. 

Currently, I am at a loss as to what I still can do. Do you have any suggestions on what I still can test? Maybe I am overlooking something simple?

Also, I have another question. When I increased the bulk density (mass per volume) and kept the fuel bed height and fuel density the same, the fire sustained better. I do not understand this, because a higher packing ratio should mean more friction right?

Also, I have been comparing my fds input file with EXMC8A of the catchpole experiment that is mentioned in the validation model. That specific experiment showed good agreement with experimental results. However, a lot of the other catchpole experiment with zero wind speed did not agree with experiments at all. Then the fire also did not progress, same as in my case. Can you maybe explain why for instance in experiment EXMC9A the fire did not progress? (only the moisture fraction was a little bit higher (7.2% instead of 6.4% for EXMC8A) and the packing ratio is four times higher (0.02 instead of 0.005 for EXMC8A).

**Comment by mcgratta** [comment]
Your initial post gives me the impression that you think there are "right" and "wrong" ways to simulate this experiment and that we, the model developers, know the right answer and we can give you a few hints to guide you. I wish it were so. It just might be that your experiment represents a particular point in an n-th degree parameter space for which the model assumptions are no longer appropriate. You might wander about this parameter space forever and never reach your goal. That is the intent of the Catchpole cases. We intentionally publish all the cases so that we can get a better sense of where there are black holes in the parameter space. 

My advice to you is to look at similar experiments and model them. Perhaps this might indicate to you the conditions for which the model assumptions break down. No wind? Really densely packed vegetation? Very sparsely packed vegetation? Moisture? Grid resolution? If you've explored all these, it just might be that the assumption that a piece of excelsior can be modeled as a sphere, cylinder, or plate is not appropriate. Or that the reaction chemistry is more complex than just a single step fast reaction. Or that the convective heat transfer correlations are not appropriate for this fuel.

Before you dive into the rabbit hole of the fundamental assumptions, however, I would try to model more than one experiment. You might have just fallen into the black hole of parameter space. Look towards that light.

**Comment by ericvmueller** [comment]
To add to what Kevin says, and similar to what Jason mentioned... the fuel mass is very low for your case. In my observations of no-wind spread in very sparse fuel beds the behavior is highly dominated by spread along individual fuel particles. This is something which I wouldn't necessarily expect the porous/dispersed media formulation to capture because that detail is all unresolved. 

While it is true that increasing the bulk density/packing ratio will increase resistance to flow, the more significant effect here is the increase in combustible mass so your flame structure and spread are no longer dominated by the details of each individual particle. That is why, in my opinion, you observe better spread when you increase bulk density and also when you compare those two Catchpole cases.

**Reply by bjmvandongen** [reply]
Thank you for your reply. It was very helpfull. 

Logically thinking, I understand that if you increase the mass per volume that you also increase the combustible mass which results in better spread. 
What I still do not understand is the following:
When I lowered the density and kept the MPV the same, the spread was also better. But when looking at the rate equations for pyrolysis for instance, the rate decreases with a lower density;  r = ρ A exp(−E/RT ). 
How can it be that the spread is better with a lower density that results in a lower reaction rate for the pyrolysis?

**Reply by bjmvandongen** [reply]
Oh and also I don't understand yet why a larger surface to volume ratio leads to better fire spread. I have only seen this parameter in the momentum equation for the drag force. But then, a higher S/V leads to more drag, which does not explain the better fire spread.

**Reply by mcgratta** [reply]
What burns more readily, a dry pine needle or a dry tree branch? What is the key parameter that distinguishes one from the other?

**Reply by bjmvandongen** [reply]
That's the surface to volume ratio. Logically thinking, I understand that. But where in the mathematical model is it incorporated?

**Reply by bjmvandongen** [reply]
I can only think of the absorption coefficient kappa.

**Reply by bjmvandongen** [reply]
The main question I have is why the fire spread is better with lower density. I can only find the density of the fuel in the reaction rates..

**Reply by ericvmueller** [reply]
Eq 17.11 the density factors into the energy equation for the solid. The surface area, and by extension surface-to-volume ratio, plays a role in the convective and radiative boundary conditions.

**Reply by bjmvandongen** [reply]
thank you very much for your response. one last question about the packing ratio. 
I read your article called: Numerical simulations of flame spread in pine needle beds using simple
thermal decomposition models

For bed4: the fuel bed height remained the same, the bulk density decreased with 25% and the porosity increased and thus the packing ratio decreased. The ROS of the fire still increased with 8% even though there was less mass available. Apperently the packing ratio also has a large influence. 
Where does the packing ratio come into the equations? I see that it comes into the equation for the drag force. A lower packing ratio results in less drag which could increase the ROS. 
But where is the packing ratio also included? or only in the equaion for the drag force?

I also see it in equation 9.10 for the absorption coefficient for radiation. But the text below also says that the absorption coefficient is not determined from eq. 9.10.

**Reply by ericvmueller** [reply]
Eq 9.10 of the Tech Guide, correct? This is referring to how frontal area density (typically 'leaf area density' in the context of a vegetative canopy) is measured in the field. But in FDS equation 9.10 is indeed used for emission and attenuation in the radiative transport equation. In fact, it is assumed the shape factor in that equation is 1/4 ...so perhaps the text could be clarified because it seems to imply one could input some effective absorption coefficient based on a light attenuation measurement, but in FDS it will always be 1/4* sigma* beta.
